fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/NavigationBar.kt

```diff
@@ -68,8 +71,10 @@ fun NavigationBar(
                         }
                     }
                     ItemType.TAB_COUNTER -> {
-                        IconButton(onClick = {}) {
-                            TabsIcon(item = it, tabsCount = 0)
+                        CompositionLocalProvider(LocalContentColor provides FirefoxTheme.colors.iconPrimary) {
```

Comment:
I am wondering why we need this?

Reply:
We need to use `LocalContentColor` because of its original usage in `TabsTray`. The tab page buttons are colored via this property, which, under the hood, uses a combination of a color and an alpha to color the selected/unselected states.

Reply:
https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt#211-261

It's a nuance of `TabRow` and `Tab` (the UI elements; not in a browser context).

Reply:
some explanation exists in the class [itself](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/compose/TabCounter.kt#43)